core/storage/classifier/gae_models_test.py

```diff
@@ -178,6 +178,23 @@ def test_create_multi_jobs(self):
         self.assertEqual(training_job2.data_schema_version, 1)
 
 
+    def test_id_collisions_exception(self):
+        next_scheduled_check_time = datetime.datetime.utcnow()
+        with self.swap(base_models, 'MAX_RETRIES', 0):
+            with self.assertRaisesRegexp(Exception, (
+                'The id generator for ClassifierTrainingJobModel '
+                'is producing too many collisions.')):
+                classifier_models.ClassifierTrainingJobModel.create(
+                    'TextClassifier', 'TextInput', 'exp_id1', 1,
+                    next_scheduled_check_time,
+                    [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+                    'state_name2', feconf.TRAINING_JOB_STATUS_NEW, None, 1)
+
+
+
+
+
+
```

Comment:
Move this few lines up too. Also a general convention to have exactly two newlines preceding a class.
Thanks!

Reply:
Ah, yes! Can you please open up an issue to track this? Thanks!

Reply:
Sure thing!
Opened #6236 in this regard. 
Thanks!